tests/models/xglm/test_modeling_tf_xglm.py

```diff
@@ -239,6 +239,7 @@ def test_batch_generation(self):
         self.assertListEqual(expected_output_sentence, batch_out_sentence)
         self.assertListEqual(expected_output_sentence, [non_padded_sentence, padded_sentence])
 
+    @slow
```

Comment:
unrelated to the original problem, but this test should be slow (uses a non-standard 0.5B model with TF) and is causing test timeouts